models/validation_error.go

```diff
@@ -0,0 +1,27 @@
+// pmm-managed
+// Copyright (C) 2017 Percona LLC
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+package models
+
+// ValidationError represents model validation error.
+type ValidationError struct {
+	msg string
+}
```

Comment:
It's required to differentiate validation errors from others. I propose to refactor existing validation functions to use it. It will help to for example https://github.com/percona/pmm-managed/blob/236bdc8a05239450d91ac6ac61359fde2f7937d7/services/server/server.go#L515

In that code, all errors will have `InvalidArgument` code, even if the problem not with the request.

Reply:
Let's do not do any of that right now – we seriously don't have time for that. The whole discussion about better validation messages and error messages (and code and other meta info) is for another time.

For now, return gRPC errors as in other existing code

Reply:
Ok, will do.

P.S. In that example that I referred we wrap `all` errors in gRPC InvalidArgument.

Reply:
Done